lib/Getopt/Long.pm

```diff
@@ -1713,6 +1713,9 @@ disable C<$verbose> by setting its value to C<0>. Using a suitable
 default value, the program can find out whether C<$verbose> is false
 by default, or disabled by using C<--noverbose>.
 
+(If both C<--verbose> and C<--noverbose> are given, the later option
```

Comment:
This is wrong. It's the last one that takes precedence, as one would expect given the other piece of new documentation

Reply:
Sorry, ambiguous wording.  By 'later' I meant whichever option is given later on the command line.